### ISSUE TITLE
Pull images from docker hub in Termux

### ISSUE THREAD

**Issue by noeckl** [indigo-dc/udocker#307]
Hello,

I want to use udocker in termux on an Android device. From an [issue](https://github.com/termux/termux-packages/issues/2755) in the termux repository, I am pretty sure it should work fine.

The installation of udocker was no problem with the following commands from the installation manual:
```
$ curl https://raw.githubusercontent.com/jorge-lip/udocker-builds/master/tarballs/udocker-1.1.7.tar.gz > $ udocker-1.1.7.tar.gz
$ export UDOCKER_TARBALL=$(pwd)/udocker-1.1.7.tar.gz
$ tar xzvf $UDOCKER_TARBALL udocker
$ chmod u+rx udocker
$ ./udocker install
```

After installation I tried pulling the hello-world image from docker hub and get the following output:
```
$ ./udocker --debug pull hello-world
get imagerepo: hello-world tag: latest
Error: in download:
header: {'X-ND-HTTPSTATUS': '', ''X-ND-CURLSTATUS': 23}
buffer:
Error: in download:
header: {'X-ND-HTTPSTATUS': '', ''X-ND-CURLSTATUS': 23}
buffer:
Error: in download:
header: {'X-ND-HTTPSTATUS': '', ''X-ND-CURLSTATUS': 23}
buffer:
v1 image id: hello-world
repo url: https://hub.docker.com/v1/repositories/hello-world/images
Error: in download:
header: {'X-ND-HTTPSTATUS': '', ''X-ND-CURLSTATUS': 23}
buffer:
Error: in download:
header: {'X-ND-HTTPSTATUS': '', ''X-ND-CURLSTATUS': 23}
buffer:
Error: in download:
header: {'X-ND-HTTPSTATUS': '', ''X-ND-CURLSTATUS': 23}
buffer:
Error: image not found or not authorized
Error: no files downloaded
```
I am not sure what to do with this error message. I would suspect that udocker uses curl internally to download images, but is not able to connect to docker hub. Has anyone had the same problem or any other insight? Any help would be very much appreciated. 

I also tried the same commands on my local machine where everything works fine.

Thank you all very much!

**Comment by jorge-lip** [comment]
Included in v1.3.0. https://github.com/indigo-dc/udocker/releases/tag/v1.3.0
Use the PROOT provided by Termux:
`UDOCKER_USE_PROOT_EXECUTABLE=$(which  proot)  ./udocker  run  mycontainer`

**Comment by noeckl** [comment]
Thank you! I am now able to download the images.
